src/HomePagePanel.js

```diff
@@ -1,81 +1,72 @@
 import React from 'react'
 import PropTypes from 'prop-types'
-
 import CardContainer from './CardContainer'
 import PanelBar from './PanelBar'
 
-import AtlasHPCard from 'atlas-homepage-cards'
-
 class HomePagePanel extends React.Component {
   constructor(props) {
     super(props)
     this.state = {
-    	panelNameA: `Species`,
-    	panelNameB: `Featured`
+    	panelNameA: this.props.panelNameAList[0],
```

Comment:
Use a descriptive name. If I read this I can’t know what is the function or purpose of `panelNameA` or `panelNameB`. What’s the content of each panel?